yarn-project/ethereum/src/deploy_l1_contracts.ts

```diff
@@ -207,8 +215,19 @@ export const deployL1Contracts = async (
 
   logger.info(`Deployed Gas Portal at ${feeJuicePortalAddress}`);
 
+  // Mock implementation of escrow takes no arguments
+  const proofCommitmentEscrow = await deployer.deploy(
+    contractsToDeploy.proofCommitmentEscrow,
+    (contractsToDeploy.proofCommitmentEscrow.contractAbi as Abi).find(
+      (fn): fn is AbiConstructor => fn.type === 'constructor',
+    )?.inputs.length === 0
+      ? []
+      : [feeJuiceAddress, account.address.toString()],
```

Comment:
A big disservice I think we did ourselves was making `feeJuice` the name for both the L1 and L2 asset.